.gitignore

```diff
@@ -6,3 +6,4 @@ VERSION
 *.rbc
 *.swp
 /.bundle
+.idea/
```

Comment:
`.idea/` auto globs `.idea/**`.  You don't need to be that specific :)
Yes, it's for Intellij IDEs.